src/_tutorials/web/fetch-data.md

```diff
@@ -400,17 +393,14 @@ When you click the button,
 the app makes a GET request of the server
 and loads the file.
 
-<aside class="alert alert-info" markdown="1">
-  **Implementation note:**
-  The original portmanteaux example loaded the co-located file `portmanteaux.json`.
-  When we moved the example into [**DartPad**]({{site.dartpad}}),
-  we couldn't co-locate the JSON file because DartPad
-  supports at most 3 files: one Dart file, one HTML file,
-  and one CSS file.
-  A workaround was to move `portmanteaux.json` to dart.dev and
-  configure dart.dev's CORS headers to allow read-only access
-  from everywhere.
-</aside>
+{{site.alert.info}}
+  **Implementation note:** The original portmanteaux example loaded the
```

Comment:
I wonder whether this note is really necessary...

Reply:
Good question. Does DartPad still have a file count (or file type) restriction? /cc @RedBrogdon @johnpryan

Reply:
Even if it does, I don't know why we explained this. We usually don't get into the specifics of how we implemented an example.

Reply:
DartPad only supports these file types right now, but it's subject to change. Maybe this could be a comment instead?

Reply:
Thinking back, I suppose we added this because we wanted to acknowledge CORS concerns. I might rewrite this note to not talk about the old implementation, because it isn't relevant now, but I guess a note might still be warranted.